packages/x-components/src/x-modules/search/components/promoted.vue

```diff
@@ -0,0 +1,74 @@
+<template>
+  <a :href="item.url" class="x-promoted" data-test="promoted">
+    <img :src="item.image" class="x-promoted__image" alt="" />
```

Comment:
For the image, in the result we have a more complex component which manages also fallback images, placeholder images, etc.

Don't we need that here? I can accept as answer that Promoted and Banners are less relevant (well, not less relevant, but they are less frequent.

Reply:
Promoted and banners must have an image:
![image](https://user-images.githubusercontent.com/6597815/125414100-b4101ca9-79c9-4e94-9a7d-eaca13741ce6.png)

Reply:
Won't do